FakeXrmEasy.Shared/FakeMessageExecutors/RetrieveCurrentOrganizationRequestExecutor.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Microsoft.Xrm.Sdk;
+using Microsoft.Crm.Sdk.Messages;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace FakeXrmEasy.FakeMessageExecutors
+{
+    public class RetrieveCurrentOrganizationRequestExecutor : IFakeMessageExecutor
+    {
+        public bool CanExecute(OrganizationRequest request)
+        {
+#if FAKE_XRM_EASY || FAKE_XRM_EASY_2013
```

Comment:
Thanks for this PR @sulzsolutions-st .  However the build is failing, I'm guessing it has to do with the precompilation symbols which will have to be moved to the top of the file, so it'll exclude messages that are not available for earlier versions entirely....